enterprise/frontend/src/embedding-sdk-bundle/components/public/CreateDashboardModal/CreateDashboardModal.tsx

```diff
@@ -3,7 +3,10 @@ import {
   getCollectionIdSlugFromReference,
   getCollectionIdValueFromReference,
 } from "embedding-sdk-bundle/store/collections";
-import type { MetabaseDashboard, SdkCollectionId } from "embedding-sdk-package";
```

Comment:
From the bundle and share modules, we won't be able to import from the package anymore, as that will stay EE